core/utils.py

```diff
@@ -60,11 +66,6 @@
 TextModeTypes = Literal['r', 'w', 'a', 'x', 'r+', 'w+', 'a+']
 BinaryModeTypes = Literal['rb', 'wb', 'ab', 'xb', 'r+b', 'w+b', 'a+b', 'x+b']
 
-# TODO(#13059): We will be ignoring no-untyped-call and no-any-return here
-# because python_utils is untyped and will be removed in python3.
-# These will be removed after python3 migration and adding stubs for new python3
-# libraries.
```

Comment:
I don't see you removing any `no-untyped-call` or `no-any-return` from this file.

Reply:
Yeah, I didn't check properly, I thought this comment was only about python_utils associated calls
So I added it back